epub33/overview/index.html

```diff
@@ -364,11 +431,18 @@ <h2>Container</h2>
 						href="https://www.w3.org/TR/epub-33/#sec-ocf">section</a> of [[EPUB-33]].</p>
 
 			</section>
-
 		</section>
+
 		<section id="sec-gls">
 			<h1>Global Language Support</h1>
 
+			<p class=ednote>
+				I wonder about this section overall. The only thing that is EPUB Specific is the section of metadata. Wouldn't it be better to 
```

Comment:
You might want to open a proper issue to consider this. I don't have a strong opinion on the GLS stuff.

Reply:
Ok. I have opened #1809, and removed the editor's note from the current version.